x-pack/plugins/rule_registry/server/utils/create_persistence_rule_type_factory.ts

```diff
@@ -79,7 +79,8 @@ export const createPersistenceRuleTypeFactory: CreatePersistenceRuleTypeFactory
                 return {
                   ...event,
                   'event.kind': 'signal',
-                  'kibana.rac.alert.id': '???',
+                  'rule.id': 'siem.customRule',
+                  'kibana.rac.alert.id': v4(),
```

Comment:
I don't see `alert.id` anywhere in the alerts as data fields. I'm wondering if this is supposed to be `kibana.alert.group.id`? No changes needed just pointing this out. I'm sure this will all change with the field migration pr.

Reply:
I reverted these changes. I know @madirey is working on these mappings and don't want to mess with them. Thanks for pointing it out.